.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
 *~
 .idea
-.vscode
\ No newline at end of file
+.vscode
+chart/skywalking/charts/
```

Comment:
Is this on purpose?

Reply:
> Is this on purpose?

Yes, this is generated locally when we try to install